waterlog/src/Pages/Usage.js

```diff
@@ -3,6 +3,8 @@ import Sensors from './../Components/Sensors';
 import Segments from './../Components/Segments';
 import SegmentsEventsList from './../Components/SegmentEventsList';
 import Segmentleaks from '../Components/Segmentleaks';
+import LeakLitres from '../Components/LeakLitres';
+import LeakCosts from '../Components/LeakCosts';
 import {Provider} from 'react-redux';
 import store from '../store';
```

Comment:
Imported objects should be uppercase (Store). Functions can be camelCase.

Reply:
Easy change because it's the default you're importing.

Reply:
Now using Upper case on imported objects.